diff --git a/src/executors/build/build.impl.js b/src/executors/build/build.impl.js
index 9feaae34c1e988bbdeed1cecb25fb220fedff81a..fea5f441702d22acd3e2c6e9501cd23f9a92bf78 100644
--- a/src/executors/build/build.impl.js
+++ b/src/executors/build/build.impl.js
@@ -19,7 +19,7 @@ async function buildExecutor(options, context) {
     // Cast to any to overwrite NODE_ENV
     process.env.NODE_ENV ||= 'production';
     const projectRoot = context.projectGraph.nodes[context.projectName].data.root;
-    (0, check_project_1.checkPublicDirectory)(projectRoot);
+    // (0, check_project_1.checkPublicDirectory)((0, path_1.join)(context.root, projectRoot));
     // Set `__NEXT_REACT_ROOT` based on installed ReactDOM version
     const packageJsonPath = (0, path_1.join)(projectRoot, 'package.json');
     const packageJson = (0, node_fs_1.existsSync)(packageJsonPath)
